shared/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.1] - 2022-09-24
+- Add unit test for `UtilityCodec()`
```

Comment:
I understand why you called in `UtilityCodec` and can see a world where we use different codecs for different modules (though it gets messy), but maybe we can rename it to `SharedCodec`? 

I think that's what through my understanding off originally.